controlpanel/frontend/views/feedback.py

```diff
@@ -0,0 +1,26 @@
+# Third-party
+from django.urls import reverse_lazy
+from django.views.generic import TemplateView
+from django.views.generic.edit import CreateView, FormMixin
+
+# First-party/Local
+from controlpanel.api.models import Feedback
+from controlpanel.frontend.forms import FeedbackForm
+from controlpanel.oidc import OIDCLoginRequiredMixin
+
+
+class CreateFeedback(OIDCLoginRequiredMixin, CreateView):
+    form_class = FeedbackForm
+    model = Feedback
+    template_name = "feedback-create.html"
+
+    def get_success_url(self):
+        return reverse_lazy("feedback-thanks")
+
+    def form_valid(self, form):
+        form.save()
+        return FormMixin.form_valid(self, form)
```

Comment:
```suggestion
```
Dont really need this as you will get this already from the `CreateView` ([this site](https://ccbv.co.uk/projects/Django/5.0/django.views.generic.edit/CreateView/#form_valid) is great for checking CBV's)